presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -29,15 +29,15 @@ For ``GRANT`` statement to succeed, the user executing it should possess the spe
 Examples
 --------
 
-Grant ``INSERT`` and ``SELECT`` privileges on the table ``orders`` to user ``alice``::
+Grant :doc:`insert` and :doc:`select` privileges on the table ``orders`` to user ``alice``::
```

Comment:
The links in synopsis are good, but I think adding this is too much and distracts from the examples.